### ISSUE TITLE
Compatibility with nice!nano and Sofle Choc

### ISSUE THREAD

**Issue by JJGadgets** [foureight84/sofle-keyboard-pimoroni#3]
Hello, I am looking to build my first split ergo, and I’m looking to build a Sofle Choc with nice!nano’s for wireless support. 

Does this mod support both the keyboard and the ZMK microcontroller? Do you know what additional work is needed to make this combo work?

**Comment by foureight84** [comment]
I have not looked at the Sofle Choc and I have not used the nice!nano. But you don't need the traces available on the PCB for this to work. You could always directly solder the Pimoroni trackball to the I2C pins, vcc and ground on the nice!nano. However, I would check with ZMK first to see their support for the Pimoroni trackball and pointing device intercommunication (depending on which side you have it connected to and assuming that ZMK also follows the master/slave design for the dual MCU communication). You could always port over the Pimoroni driver and add modify the intercommunication library to support it (this is just my assumption).

TLDR; I am not sure about the firmware support. You'll have to dive deeper in the ZMK code or ask the community.
